packages/core-data/src/types/README.md

```diff
@@ -0,0 +1,238 @@
+# Entity Records Types
```

Comment:
This document is really great :tada: Can you add a link to it from the package's README?

Reply:
I just did:

Core data is shipped with [`TypeScript definitions for WordPress data types`](/packages/core-data/src/types/README.md).